LeaderboardBackend/Controllers/Requests/PasswordAttribute.cs

```diff
@@ -0,0 +1,65 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace LeaderboardBackend.Controllers.Requests
+{
+	public class PasswordAttribute : ValidationAttribute
+	{
+		private static readonly int MIN = 8;
+		private static readonly int MAX = 80;
+
+		public string GetErrorMessageLength(bool tooShort)
+		{
+			if (tooShort) {
+				return $"Your password needs to be at least {MIN} characters long.";
+			}
+			return $"Your password needs to be at most {MAX} characters long.";
+		}
+
+		public string GetErrorMessageMissing(List<string> missing)
+		{
+			if (missing.Count == 1) {
+				return $"Your password still needs {missing[0]}.";
+			}
+			if (missing.Count == 2) {
+				return $"Your password still needs {missing[0]} and {missing[1]}.";
+			}
+			return $"Your password still needs {String.Join(", ", missing.GetRange(0, missing.Count - 1).ToArray())}, and {missing[missing.Count - 1]}";
+		}
+
+		protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
+		{
+			var user = (LoginRequest)validationContext.ObjectInstance;
```

Comment:
This PR has existed longer than this decision so obviously you couldn't have known for this PR, but I think we're moving away from using `var` at all.

Reply:
Oh yeah. Gonna update everything.